src/Ghcid.hs

```diff
@@ -216,15 +218,15 @@ runGhcid session waiter termSize termOutput opts@Options{..} = do
 
             let updateTitle extra = unless no_title $ setTitle $
                     let f n msg = if n == 0 then "" else show n ++ " " ++ msg ++ ['s' | n > 1]
-                    in (if countErrors == 0 && countWarnings == 0 then allGoodMessage else f countErrors "error" ++
+                    in (if countErrors == 0 && countWarnings == 0 then (allGoodMessage ++ " at " ++ currTime) else f countErrors "error" ++
```

Comment:
remove these brackets to get hlint happy 👍

Reply:
Sure!